### DISCUSSION TITLE
Upscayl 2.9.9 Decoding Error

### DISCUSSION THREAD

**Discussion by gabetom24** [upscayl/upscayl#748] [Q&A] [answered]
When I start Upscaling and click to go to another app of any kind this error pops up. I have checked if it is using the right GPU and I have also checked if my GPU supports Vulkan. this is not a big issue; leaving the app open will let me upscale images. Still, I have come to find out that if I use a second monitor and leave the app open on one monitor I can still interact with other apps on another monitor.
I am using a GTX 1050 (MOBILE) GPU
![Screenshot 2024-03-11 093545](https://github.com/upscayl/upscayl/assets/91073322/dfe9a37a-a202-4257-807c-502ecff02c83)

**Comment by aaronliu0130** [comment, marked as answer]
Please follow the bug reporting guide.
